### ISSUE TITLE
Suggestion: Add `focus-view <stack-index-integer>`

### ISSUE THREAD

**Issue by Marenz** [riverwm/river#668]
I understand that the views are kept in a list and no further information is available. However the layout-generate does have all that info and could implement its own "up|down" etc commands that would map to the specific index in the view stack and use the suggested command to focus it directly.

Maybe running `focus-view next|previous`  in quick succession can reach the same thing.. I haven't tried it, though I'd be worried it might look strange for the user when it quickly focuses several views until it reaches the target view...

**Comment by Leon-Plickat** [comment]
I like the general idea, but since layout generators don't know which view is focused, this wouldn't quite work.

**Comment by Marenz** [comment]
Right, but couldn't a layout generator not use `river-status-unstable-v1` to get that information?

**Comment by Leon-Plickat** [comment]
No. That protocol only allows you to get the title of the focused view.

**Comment by Marenz** [comment]
Ah, then I misunderstood the wiki description. I suppose having a way to get the focus would be my follow-up suggestion :D

**Comment by MaxVerevkin** [comment]
> but since layout generators don't know which view is focused

Is there a specific reason why this is the case? This information would allow for some nice layouts, for example a maser-stack where the focused view in the stack would have a larger height than other stack views.

This can be implemented in a backwards-compatible way similar to #671. If no one is against this I may try to implement it.
